_posts/2020-02-26-flink-for-data-warehouse.md

```diff
@@ -0,0 +1,108 @@
+
+To take it a step further, Flink 1.10 introduces [compatibility of Hive built-in functions via HiveModule](https://ci.apache.org/projects/flink/flink-docs-release-1.10/dev/table/hive/hive_functions.html#use-hive-built-in-functions-via-hivemodule). Over the years, Hive community has developed a few hundreds of built-in functions that are super handy for users. For those built-in functions that don't exist in Flink yet, users would be able to leverage Hive built-in functions that they are familiar of and complete their jobs seamlessly.
+
+
+### Enhanced Read and Write on Hive Data
+
+Flink 1.10 extends its read and write capabilities on Hive data to all the common use cases with better performance. 
+
+On the reading side, Flink now can read Hive regular tables, partitioned tables, and views. Lots of optimization techniques are developed around reading, including partition pruning and projection pushdown to transport less data from file storage, limit pushdown for faster experiment and exploration, and vectorized reader for ORC files.
+
+On the writing side, Flink 1.10 introduces “INSERT INTO” and “INSERT OVERWRITE” to its syntax, and can write to not only Hive’s regular tables, but also partitioned tables with either static or dynamic partitions.
+
+### Formats
+
+Your engine should be able to handle all common types of file formats to give you the freedom of choosing one over another in order to fit your business needs. It’s no exception for Flink. We have tested the following of table storage formats: text, csv, SequenceFile, ORC, and Parquet.
+
+### More Data Types
+
+In Flink 1.10, we added support for a few more frequently-used Hive data types that were not covered by Flink 1.9. Flink users now should have a full, smooth experience to query and manipulate Hive data from Flink.
+
+### Roadmap
+
+Integration between any two systems is a never-ending story. 
+
+We are constantly improving Flink itself and the Flink-Hive integration also gets improved by collecting user feedback and working with folks in this vibrant community.
+
+After careful consideration and prioritization of the feedback we received, many of the below requests, together with other improvements, have been prioritized for the next Flink release, Flink 1.11.
+
+- Native Parquet reader for better performance
+- Additional interoperability - support creating Hive tables, views, functions in Flink
+- Better out-of-box experience with built-in dependencies, including documentations
+- JDBC driver so that users can reuse their existing toolings to run SQL jobs on Flink
+- Hive streaming sink so that Flink can stream data into Hive tables, bringing real streaming experience to Hive
+- Hive syntax and semantic compatible mode, so that users 
```

Comment:
@bowenli86 

I think there is something missing here? 
_Hive syntax and semantic compatible mode, so that users [.....]_ 

Should the "so that users" be deleted? or should we add something more here?